docs/developers-clear-containers-install.md

```diff
@@ -36,8 +36,9 @@ Since the installation guide will have installed packaged versions of
 all required components, it is only necessary to install the source for
 the component(s) you wish to develop with.
 
-**IMPORTANT:** Do not combine [Clear Containers 2.1](https://github.com/01org/cc-oci-runtime) and [Clear Containers 3.0](https://github.com/clearcontainers).
-Both projects ship ``cc-proxy`` and they are not compatible with each other.
+**IMPORTANT:** Do not combine [Clear Containers 2.x](https://github.com/01org/cc-oci-runtime) and [Clear Containers 3.x](https://github.com/clearcontainers) on the same system.
```

Comment:
Thanks for doing those @jodh-intel  I've been trying to push the 'n.x' notation to reduce the number version rot...
There are places where hard versions probably do make sense (some install or arch docs for instance might be specific) - but they are probably less common than we currently carry in the docs.